src/wazuh_modules/wm_fluent.c

```diff
@@ -607,9 +607,8 @@ static int wm_fluent_handshake(wm_fluent_t * fluent) {
         }
 
         if (helo->auth_size > 0) {
```

Comment:
According to the protocol, receiving an auth message does not imply that you have to provide a non-empty user/password and it should not alert if they are empty.
Moreover, given that now credentials are always filled, this debug message will always trigger.